deployment/default.nix

```diff
@@ -5,13 +5,18 @@ let
   terraform = (callPackage ./terraform.nix { }).terraform_0_12;
 
   getCreds = pkgs.writeShellScript "getcreds" ''
+    if [[ $1 == "" || $2 == "" ]]; then
```

Comment:
The standard way of doing this would be `if [[ $# -ne 2 ]]` (where `$#` means number of arguments).

Reply:
I'm not ashamed that I didn't know that

Reply:
No need to be. :-)